classes/services/PKPStatsService.inc.php

```diff
@@ -570,4 +569,13 @@ private function _getTimeSegments($startDate, $endDate, $timeSegment) {
 		return $timeSegments;
 	}
 
+	public function getUserStatistics($contextId, $args = array()) {
+		return \DAORegistry::getDAO('MetricsDAO')
+			->getUserStatistics($contextId, $args['dateStart'] ?? null, $args['dateEnd'] ?? null, $args['timeSegment'] ?? null);
```

Comment:
The third parameter here and below (`timeSegment`) are forgotten -_-
I was using it to group the averages by day/month/year, but I've decided to drop it in order to reduce the code...

Reply:
Yeah, it's already removed from the MetricsDAO, just need to clean this.